data/remnant missions.txt

```diff
@@ -1506,3 +1510,150 @@ mission "Remnant: Broken Jump Drive job"
+			`	As you are leaving Plume's lab a technician pulls you aside. "Our records mention that the tentacles had some interesting properties that were useful in combat. If you are interested, meet me in the spaceport in a few hours."`
+
+
+
+mission "Remnant: Face to Maw 2"
+	name "Take the slime sample to Viminal"
+	description "The slime has properties of interest to a team of engineers on <planet>."
+	to offer
+		has "Remnant: Face to Maw 1: done"
+	source "Aventine"
+	destination "Viminal"
+	cargo "slime sample" 1
+	on offer
+		conversation
+			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a datapad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that" he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though."`
+			`	"Be that as it may be, the more immediate reward for your curiosity is in this sample," he trills with excitement. "Several of our labs on Viminal have been working on the development of a new generation of ship technology, and this substance contains some chemicals that could be a key component. I have already transmitted the data to them, but they would like to compare with the original. Could you take this sample to them?"`
+			choice
+				`	"Yes, I can."`
+					accept
+				`	"I would rather not."`
+			`	Plume stoically accepts that with a nod "That is your choice. I will arrange for another ship to transport it."`
+			decline
+	on accept
+		event "remnant: penguin"
+	on complete
+		log "People" "Tali" `As an engineering prefect, Tali is the foremost authority among the Remnant on salvaged technology and a senior leader in their reverse engineering program. She is also a prominent engineer in the field of spacecraft propulsion.`
+		"remnant tali" ++
+		payment 1000000
+		conversation
+			branch tali
+				"remnant tali" > 1
+			`When you land on Viminal a solidly-built female Remnant in coveralls is waiting for you. "Greetings, Captain <first>. My name is Tali, and I am a Prefect among the Remnant. Above all else, I study motion, and in particular how our starships move."`
```

Comment:
![31b-me2-021717](https://user-images.githubusercontent.com/1097249/55294519-55cf8380-53c0-11e9-823c-9207f721e1d4.jpg)

Female engineer named Tali who's fascinated with starships == an existing Mass Effect character. She should probably have a different name. :)

Reply:
>She should probably have a different name. :)

We can have references? :)
There was a conversation about it in #3761.

Starts here: https://github.com/endless-sky/endless-sky/pull/3761#discussion_r243917477

Reply:
Yeah but this isn't a reference, it's a straight-up rip-off. :) I mean you wouldn't create a character named Darth Vader who dresses in all black, wears a mask, and leads the enemy forces, would you?

Reply:
Idk, does this Darth Vader also carry around a laser sword, can use magic to effect things at a distance, and is the father of the main character? I've never played Mass Effect, but I'm sure there's more to Tali's character aside from being female and into spaceships.

Reply:
Name, female, engineer, and focused on starships is about the extent of the similarities.

In terms of personality, Mass Effect Tali starts out quite young, shy, usually reserved, and is someone just starting out in the world, trying to make a name for herself. Later in the series she develops some self-confidence and becomes a leader. She often flirts with the protagonist (and is one of the characters the player can become romantically involved with). In-game she is what I'd call a "tweaker engineer" - as in, she spends most of her time adjusting, monitoring, and fine-tuning things. Which fits with the Quarian culture, as they are a ship-bound nation that doesn't have any planets or easy access to any kind of resources. So it is rare that they build anything new, and they spend most of their time maintaining and tweaking their current ships - most of which are many centuries old.

Remnant Tali starts out as a confident, mature person who is already well established in terms of her position and abilities. She has her own interests and life, which do not revolve around the protagonist, and is not flirtatious in the least. In-game, she is portrayed as being  a hands-on inventor: Building ships, installing new things, designing new ships, etc. The Remnant have access to three planet's worth of resources, including ones with functioning advanced alien technology, rare crystals, and other advantages. They are described as being fast innovators who managed to develop and build new warships based on the alien tech they found in very short order, so being  a designer and builder fits better.

Physically speaking, Mass Effect Tali has a slight, curvaceous silhouette, and the player never sees anything underneath that environment suit. ME Tali is effectively immuno-compromised (hence the suit), is physically weak, and not particularly hardy. In-game, Tali is most often found leaning over control consoles, pondering the read-outs. She is also quite young, and is described as being in the first few years of adult-hood. When the player initially meets her, she is in the midst of her pilgrimage - a rite of passage to demonstrate that she is an adult and prove her worth to society. Even by the end of the series (which at most is 3-4 years later) she is still one of the youngest Quarians that the player interacts with.

Remnant Tali is described physically as being a sturdy and solidly-built individual who is comfortable in short sleeves even in the near freezing temperatures of Viminal. In-game, Tali is usually found in some sort of physically active scene - climbing over ships, installing components, tightening things with wrenches, etc. She is an adult approaching mid-life: Still very physically fit, but isn't brash about it the way younger people often are.

---

In short, the Tali in here is a deliberate tribute to the Mass Effect Tali, but one that is intentionally designed to chart her own path with her own identity in terms of both her personality and physicality. For players that see the name and think Tali from ME, I intend her to be memorable in how she will clash with their expectations, and that will include descriptions of her doing things that the ME Tali never did. Basically, taking the player's expectation of finding the slim vulnerable Tali from ME, and instead finding what amounts to Endless Sky's equivalent of Rosie the Riveter.


![steampunk-rosie-the-riveter-we-can-do-it](https://user-images.githubusercontent.com/32169904/55670443-8771a000-5841-11e9-9367-98f3fa3df6f5.png)
(Not the original one, I know, but it was the closest thing I could find searching for "Sci Fi Rosie the Riveter")

Reply:
I get what you're doing, I just don't think it works. Whenever I read the name I'm unable to avoid my brain jumping to ME Tali.

Reply:
Your Tali will always be the ME Tali to me too, regardless of how much you try to push the differences.

Reply:
I get that.

Should I change the spelling, then? Talie, perhaps?

Reply:
OK, all instances of "Tali" have been replaced with "Taly". Does that work?

Reply:
@Pointedstick @jafdy One thing to point out, however. Tali's already in the story as Tali for a few months now. How many saves will that potentially confuse? It won't break any saves or progression, but it might trigger an extra "haven't met Taly before" branch and an extra log entry.

Reply:
Non-live version, I think it'll be fine. (And I would like to say that I objected to this in the first instance.)

Reply:
Yes, and if I recall correctly, ended up getting battered into submission by a variety of people saying it'll be fine. But if after a few months of ES having a tali too, you are still associating Tali with a frail immune-compromised alien instead of a strong dark skinned engineer, well, time to change it before too many people get used to it.

Reply:
#4254 Should solve this for you.